fails:Hash#merge returns a new hash by combining self with the contents of other
fails:Hash#merge sets any duplicate key to the value of block if passed a block
fails:Hash#merge tries to convert the passed argument to a hash using #to_hash
fails:Hash#merge does not call to_hash on hash subclasses
fails:Hash#merge returns subclass instance for subclasses
fails:Hash#merge processes entries with same order as each()
fails:Hash#merge! adds the entries from other, overwriting duplicate keys. Returns self
fails:Hash#merge! sets any duplicate key to the value of block if passed a block
fails:Hash#merge! tries to convert the passed argument to a hash using #to_hash
fails:Hash#merge! does not call to_hash on hash subclasses
fails:Hash#merge! processes entries with same order as merge()
fails:Hash#merge! raises a RuntimeError on a frozen instance that is modified
fails:Hash#merge! checks frozen status before coercing an object with #to_hash
fails:Hash#merge! raises a RuntimeError on a frozen instance that would not be modified
fails:Hash#merge! does not raise an exception if changing the value of an existing key during iteration
